common/app/model/content/Atoms.scala

```diff
@@ -53,6 +54,11 @@ final case class MediaAtom(
       "^0".r.replaceFirstIn(formattedDuration, "") //strip leading zero
     }
   }
+
+  def activeAsset: MediaAsset = activeVersion match {
+    case Some(version) => assets.find(_.version == version).getOrElse(assets.head)
+    case _ => assets.head
+  }
```

Comment:
Ok but what if the `activeAsset` function called by a piece of code that doesn't make the check beforehand?

I think it is better to avoid any `exception` to be thrown, make the "error" case explicit and ensure the "error" case is handled at the edge when something needs to be rendered.

I would keep it simple and display nothing if `assets` is empty but we can consider another solution if you prefer. 

What do you think?